Sources/StreamChatUI/ChatMessageList/ChatChannelVC.swift

```diff
@@ -99,4 +99,27 @@ extension _ChatChannelVC: _ChatChannelControllerDelegate {
     ) {
         messageList.updateMessages(with: changes)
     }
+
+    public func channelController(
+        _ channelController: _ChatChannelController<ExtraData>,
+        didChangeTypingMembers typingMembers: Set<_ChatChannelMember<ExtraData.User>>
+    ) {
+        guard let currentMember = channelController.channel?.membership else {
+            typingIndicatorView.setAnimatedly(hidden: true)
+            log.error("Current member not found in ChatChannelVC.")
+            return
+        }
+
+        let typingMembersWithoutCurrentUser = typingMembers.subtracting([currentMember])
+
+        if typingMembersWithoutCurrentUser.isEmpty {
+            typingIndicatorView.setAnimatedly(hidden: true)
+        } else if typingMembersWithoutCurrentUser.count == 1 {
+            guard let member = typingMembers.first, let name = member.name else { return }
+            typingIndicatorView.content = "\(name) " + L10n.MessageList.TypingIndicator.typingSingular
+        } else {
+            guard let member = typingMembers.first, let name = member.name else { return }
```

Comment:
Same here

Reply:
Extracted the code above, however question is if `Someone is typing` in DM Channel isn't strange 😅